biapol_utilities/data/_data.py

```diff
@@ -25,6 +25,24 @@ def blobs():
     return io.imread(os.path.join(data_dir, "blobs.png"))
 
 
+def hourglass():
+    """Gray-level 4D hourglass image [1].
+
+    Can be used for polar transformations, tracking, dimentionality reduction.
+
+    Returns
+    -------
+    hourglass : (50, 100, 200, 200) uint8 ndarray
+        Hourglass 3D+t image depicting an unrealisitc bright half-circle
+        droplet sliding through the glass wall.
+
+    References
+    ----------
+    .. [1] Created with a Fiji macro by Marcelo Leomil Zoccoler
```

Comment:
Hi Marcelo @zoccoler ,

just a quick question: Would it be possible to rewrite this macro as python script and put it here instead of a 200 MB file? Keep in mind: Whenever somebody downloads our library from the internet, e.g. via pip, they would have to download 200 MB...

Reply:
Sure :)
Should I put the script within `_data.py`?

Reply:
Yes! I'd suggest you put the python code here in this python function so that `hourglass()` still returns the same 200 MB data, but instead of loading the data, it's generated.

Reply:
Ok, it made sense in my mind to design the hourglass as an object, which can produce an image. I added both to the data module.
I reversed the commit where I had uploaded the big stack.